### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by basileLeroy** [SezinAkbulut/accessible-website#1]
# Feedback

Great website, it looks very good!

There are a couple of accessibility issues that I would like to remark:
- Make sure to always use a correct order when using `headings`. Don't go from `<h1>` to `<h3>` for example, 
- Buttons should always content some form of text. I can spot a couple of buttons that only contain an SVG or an icon. but those are not readable for screen readers.
- I like the map you made in the home page, but be carefull, your "pin" icons are focussable, but don't show any styling when in focus!

About contrast and text:
- When zoomed in to 200%, some of your text becomes stuck behind other content. 
- When zoomed in to 200%, your text can overlap with images sometimes. the contrast with the image in the background is not ideal for accessibility.

**Use the lighthouse and the Axe DevTools extensions to check on what you can improve even more**

## Git
Generally, your commits are ok!

But there is one thing that I would like to mention:
- Check for spelling/grammar before committing: I can see a lot of spelling/grammar errors, make sure to commit correct sentences
